org-formation/600-access/_tasks.yaml

```diff
@@ -1,4 +1,20 @@
 
+JumpcloudContainerCentralAdmin:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/IAM/jumpcloud-idp.yaml
+  DependsOn: CommunityIamSamlProviderRP
+  StackName: jc-access-containercentral-admin
+  MaxConcurrentStacks: 10
```

Comment:
What does this max constraint mean?

Reply:
it means deploy 10 of these stacks concurrently, in case you want to deploy to all regions.  here's the doc.. https://github.com/org-formation/org-formation-cli/blob/master/docs/task-files.md#task-types